packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.base.tsx

```diff
@@ -206,13 +204,16 @@ export class CoachmarkBase extends BaseComponent<ICoachmarkProps, ICoachmarkStat
       isMeasured
     } = this.state;
 
+    // Defaulting the main background before passing it to the styles because it is used for `Beak` too.
+    const deafultColor = color || theme!.semanticColors.primaryButtonBackground;
```

Comment:
Seen that it's better practice to do if (theme) checks instead of using the ! operator - suggest removing that bang usage here.

Reply:
Fixed!